go/vt/vttablet/tabletserver/schema/historian.go

```diff
@@ -163,8 +169,17 @@ func (h *historian) loadFromDB(ctx context.Context) error {
 		return err
 	}
 	defer conn.Recycle()
-	tableData, err := conn.Exec(ctx, sqlparser.BuildParsedQuery(getSchemaVersions, sidecardb.GetIdentifier(),
-		h.lastID).Query, 10000, true)
+
+	var tableData *sqltypes.Result
+	if h.lastID == 0 && h.schemaMaxAgeSeconds > 0 { // only at vttablet start
+		schemaMaxAge := time.Now().Add(time.Duration(-h.schemaMaxAgeSeconds) * time.Second)
```

Comment:
What about timezone related issues? I think that we should explicitly use UTC here.